scripts/lua/modules/alert_definitions/host/host_alert_domain_names_connection.lua

```diff
@@ -0,0 +1,64 @@
+--(C) 2019-21 - ntop.org
+
+
+--##############################################
+
+local host_alert_keys = require "host_alert_keys"
+package.path = dirs.installdir .. "/scripts/lua/modules/?.lua;" .. package.path
+local alert_creators = require "alert_creators"
+local json = require("dkjson")
+--Import the classes library.
+local classes = require "classes"
+--Make sure to import the Superclass!
+local alert = require "alert"
+
+--##############################################
+
+local host_alert_domain_names_connection = classes.class(alert)
+
+--##############################################
+
+host_alert_domain_names_connection.meta = {
+alert_key = host_alert_keys.host_alert_domain_names_connection,
+i18n_title =  "alerts_dashboard.threashold_cross",
+icon = "fas fa-fw fa-arrow-circle-up",
+
+}
+
+--##############################################
+
+--@brief Prepare an alert table used to generate the alert
+--@param one_param The first alert param
+--@param another_param The second alert param
+--@return A table with the alert built
+function host_alert_domain_names_connection:init(metric, value, operator, threshold)
+--Call the parent constructor
+self.super:init()
+
+self.alert_type_params = alert_creators.createThresholdCross(metric, value, operator, threshold)
```

Comment:
This is not needed, as the json is generated from the C++ alert class.

Reply:
Do you mean that we have to delete whole "function host_alert_domain_names_connection:init([params])" or use that function without params and "self.alert_type_params = {}" in its body?

Reply:
just leave the `self.super:init()` in the method. That's enough. You are populating `alert_type_params` in C++